test/angular_index_test.py

```diff
@@ -208,6 +208,7 @@ def test_only_one_item(self):
         self.assertEquals(idx.get_n_items(), 1)
         self.assertEquals(idx.get_nns_by_vector(vector=numpy.random.randn(100), n=50, include_distances=False), [0])
 
+    @SkipTest # TODO: fix crash
```

Comment:
is there an easy way for you to figure out why it's crashing? it doesn't crash on linux and in appveyor

Reply:
Will be trying the VS mixed-mode debugger this weekend. From additional logging the crash occurs in https://github.com/spotify/annoy/blob/a7b351222cc0ab25ac1ecd6282c1d26836d3a003/src/annoylib.h#L932 when ``indices.size() == 0``. I can't imagine why as that sounds equivalent to a nop, but conditionally executing the memcpy fixes the test.

Reply:
maybe just doing a `if(!indices.empty())` gate would resolve it on all platforms

weird though

Reply:
oh i bet `indices[0]` will cause a memory access violation if indices has zero size. that's funny.

easy fix though. i can do it myself.

Reply:
Yes that's what I used. The &index[0] should be fine as we only ever get the address, so I suspect a bug in this old memcpy implementation may be causing an unnecessary attempt to read